spec/requests/api/graphql/queries/offer_spec.rb

```diff
@@ -65,7 +65,7 @@
         expect(offer_response).to eq nil
 
         error_message = body['errors'][0]['message']
-        expect(error_message).to eq "Can't access offer"
+        expect(error_message).to eq 'Offer not found'
```

Comment:
Changed the message here in order to not leak that an offer exists if you're not authorized to view it.